index.html

```diff
@@ -30,11 +30,12 @@
 </head>
 <body>
   <header class="section masthead" id="masthead">
-      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 578.55 185.54" class="mozilla-logo">
-        <title>Mozilla</title>
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 578.55 185.54" class="mozilla-logo" xmlns:xlink="http://www.w3.org/1999/xlink">
+      <a xlink:href="https://www.mozilla.org/"        <title>Mozilla</title>
```

Comment:
This line is missing a closing `>`. It would be good to move the `<title>` to the next line as well.

Reply:
In fact, I'm tempted to suggest we do `<a href="..."><svg>...</svg></a>` instead of putting the `<a>` inside of the `<svg>` element. That should avoid the need for the extra XML gunk that makes it harder to read.

Reply:
Ok. I believe I made those changes in the newest pull request.